Description: Trivial patch to mention new config files under "man nullmailer"
Author: Nick Leverton <nick@leverton.org>

Index: nullmailer-1.13/doc/nullmailer.7
===================================================================
--- nullmailer-1.13.orig/doc/nullmailer.7	2014-07-31 21:25:15.000000000 +0100
+++ nullmailer-1.13/doc/nullmailer.7	2014-07-31 21:25:37.000000000 +0100
@@ -23,9 +23,11 @@
 .I adminaddr	\fBnullmailer-queue
 .I defaultdomain	\fBnullmailer-inject
 .I defaulthost	\fBnullmailer-inject
+.I helohost	\fBnullmailer-send
 .I idhost	\fBnullmailer-inject
 .I me		\fBnullmailer-inject
 .I pausetime	\fBnullmailer-send
 .I remotes	\fBnullmailer-send
+.I sendtimeout	\fBnullmailer-send
 .fi
 .RE
